README.rst

```diff
@@ -18,7 +18,7 @@ The system is built using a modular approach so support for other devices or act
 components <https://developers.home-assistant.io/docs/creating_component_index/>`__.
 
 If you run into issues while using Home Assistant or during development
-of a component, check the `Home Assistant help section <https://home-assistant.io/help/>`__ of our website for further help and information.
+of a component, Please check out `Home Assistant help section <https://home-assistant.io/help/>`__ of our website for further help and information.
```

Comment:
The capitalization here isn't correct. Besides that we don't need to add these two words here and adding two surplus words to our Readme isn't the kind of PR we want to see during Hacktoberfest.